src/test/scala/cognite/spark/v1/AssetsRelationTest.scala

```diff
@@ -5,25 +5,25 @@ import io.scalaland.chimney.dsl._
 import org.apache.spark.sql.{DataFrame, Row}
 import org.apache.spark.SparkException
 import org.apache.spark.sql.functions._
-import org.scalatest.{FlatSpec, Matchers}
+import org.scalatest.{FlatSpec, Matchers, ParallelTestExecution}
 
 import scala.util.control.NonFatal
 
-class AssetsRelationTest extends FlatSpec with Matchers with SparkTest {
+class AssetsRelationTest extends FlatSpec with Matchers with ParallelTestExecution with SparkTest {
 
   val sourceDf = spark.read
     .format("cognite.spark.v1")
     .option("apiKey", readApiKey)
     .option("type", "assets")
     .load()
-  sourceDf.createTempView("sourceAssets")
+  sourceDf.createOrReplaceTempView("sourceAssets")
```

Comment:
I don't see why is this necessary? Would it be better to use `createTempView` to prevent races?

Reply:
If the same SparkSession somehow gets reused the `createTempView` will fail, and the tests abort. Typically that shouldn't happen, but I've seen it, and in that case we do want to just replace the existing one.